src/plugins/controls/public/options_list/components/options_list_popover_suggestions.tsx

```diff
@@ -159,6 +159,13 @@ export const OptionsListPopoverSuggestions = ({
       <div ref={listRef}>
         <EuiSelectable
           options={selectableOptions}
+          renderOption={(option) => {
+            return (
+              <EuiHighlight search={option.key === 'exists-option' ? '' : searchString.value}>
+                {option.label}
+              </EuiHighlight>
```

Comment:
I don't think it makes sense for the "Exists" option to be highlighted, even when the search string matches:

| Without this "Exists" check | With this "Exists" check |
|--------|-------|
| ![image](https://github.com/elastic/kibana/assets/8698078/d560bd81-ea64-4a80-853a-ef43fe7f1d60) | ![image](https://github.com/elastic/kibana/assets/8698078/e7cff157-5e5e-43bc-b60d-e66fc626413f) |

After all, the "Exists" option shows up **regardless** of if the search string matches it or not, so it doesn't really make sense to only sometimes highlight the substring for this option:

![image](https://github.com/elastic/kibana/assets/8698078/5b3395d6-2543-4119-9ea6-bf14e9366a58)